packages/website/src/components/team/TeamBio.module.css

```
@@ -0,0 +1,47 @@
+.teamBio {
+  --square-size: 9rem;
+
+  display: flex;
+  flex-direction: column;
+  max-width: var(--square-size);
+}
+
+.profilePhoto {
+  width: var(--square-size);
+  height: var(--square-size);
+}
+
+.texts {
+  display: flex;
+  gap: 0.5rem;
+  margin-top: 0.75rem;
+  flex-direction: column;
```

[Very Nit, Stylistic] 

Putting related props closer might make it easier to read (grouped the flex props)

```suggestion
  display: flex;
  flex-direction: column;
  gap: 0.5rem;
  margin-top: 0.75rem;
```

For some reason, this feels more chronological to me, WDYT?

---

Thanks for the reminder - I actually have a moderate person preference for alphabetical.... 😄